br/pkg/task/stream.go

```diff
@@ -454,13 +454,8 @@ func (s *streamMgr) backupFullSchemas(ctx context.Context, g glue.Glue) error {
 	return nil
 }
 
-func (s *streamMgr) checkStreamStartEnable(g glue.Glue) error {
-	se, err := g.CreateSession(s.mgr.GetStorage())
```

Comment:
The function `CreateSession` doesn't call `session.InitMDLVariable`. If it skips creating domain in mgr, creating index would be stuck in state `write only` or `delete only`.